packages/contracts/test/dollar/BondingCurve.t.sol

```diff
@@ -0,0 +1,135 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.3;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "../helpers/LiveTestHelper.sol";
+
+contract ZeroState is LiveTestHelper {
+
+    event Deposit(
+        address indexed user, 
+        uint256 amount
+    );
+
+    event Withdraw(
+        uint256 amount
+    );
+
+    uint32 connectorWeight;
+    uint256 baseY;
+    uint256 constant ACCURACY = 10e18;
+    uint256 public constant BONDING_TOKEN_ID = 1;
+    address public treasuryAddress;
+
+    mapping (address => uint256) public share;
+
+    function setUp() public virtual override {
+        super.setUp();
+    }
+}
+
+contract RemoteZeroStateTest is ZeroState {
+
+    function testCannotDeployEmptyUbiquistickAddr() public {
+        vm.expectRevert("NFT address empty");
+        BondingCurve broken = new BondingCurve(
+            address(manager),
+            address(0)
+        );
+    }
+
+    function testCannotDeployConnectorWeightZero() public {
+        vm.expectRevert();
+        BondingCurve broken = new BondingCurve(
+            address(manager),
+            address(0)
+        );
+    }
+
+
+    uint256 connWeight = bound(uint256(connectorWeight), 1, 1000000);
```

Comment:
to make fuzzing works it should be parameters of the functions like 
```

    function testSetParams(uint256 connectorWeight, uint256 baseY) public {
        uint256 connWeight = bound( connectorWeight , 1, 1000000);
        uint32 _connectorWeight = uint32(connWeight);
        uint256 _baseY = bound(baseY, 1, 1000000);
        vm.prank(admin);
        bondingCurve.setParams(
            _connectorWeight,
            _baseY
        );

        assertEq(bondingCurve.connectorWeight(), _connectorWeight);
        assertEq(bondingCurve.baseY(), _baseY);
    }

```

Reply:
Done

Reply:
where is it done ? Did you  push already ?